packages/browser/src/index.ts

```diff
@@ -86,6 +86,47 @@ export default class LogtoClient {
     sessionStorage.setItem(this.logtoSignInSessionKey, jsonItem);
   }
 
+  public isAuthenticated() {
```

Comment:
better to use property getter here?

Reply:
I think both are fine, but I don't see one better than the other. And since the other three getter functions all have parameters, they are not allowed to use property getter pattern.
Therefore, for the sake of consistency, we should all define it in the normal function pattern?

Reply:
from the naming it's like a prop actually, unless it's called `getIsAuthenticated` or something

Reply:
Updated, thanks